packages/contracts/test/helpers/test-runner/test-runner.ts

```diff
@@ -217,6 +224,12 @@ export class ExecutionManagerTestRunner {
 
     this.contracts.OVM_DeployerWhitelist = DeployerWhitelist
 
+    this.contracts.OVM_ProxyEOA = await getContractFactory(
```

Comment:
Here we deploy the reference version of the `OVM_ProxyEOA` which is duplicated by the `OVM_ExecutionManager`.